sdk/storage/Azure.Storage.Internal.Avro/src/Azure.Storage.Internal.Avro.csproj

```diff
@@ -7,7 +7,8 @@
     <Version>12.0.0-preview.1</Version>
     <GenerateAPIListing>false</GenerateAPIListing>
   </PropertyGroup>
-  <ItemGroup>
+  <!-- These packages are included as part net6.0; the external references are not needed. -->
+  <ItemGroup Condition="'$(TargetFramework)' != 'net6.0'">
```

Comment:
For maintainability should this be inverted so that adding future frameworks doesn't need the conditions to be touched?